build.gradle.kts

```diff
@@ -431,7 +431,7 @@ subprojects {
     reports.html.outputLocation.set(file("${rootProject.projectDir}/build/reports/"))
     val skipTests = project.hasProperty("skipTests")
     if (!skipTests) {
-      jvmArgs = listOf("-Xmx2G")
+      jvmArgs = listOf("-Xmx3G")
```

Comment:
Is 3G enough to solve the OOM problem?

Reply:
I think it's enough, if not we can increase the size.

Reply:
Seems not enough, CI failed in this PR

Reply:
Change to 4G